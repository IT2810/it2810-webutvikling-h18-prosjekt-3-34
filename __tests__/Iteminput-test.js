import React from "react";
import Iteminput from ".././components/iteminput";

import renderer from "react-test-renderer";

/**
 * Test klasse for iteminput komponent
 */

// 1. Snapshot test
test("renders correctly", () => {
  const tree = renderer.create(<Iteminput />).toJSON();
  expect(tree).toMatchSnapshot();
});

// Metoden addItem bruker bare metoder lengre opp i hierarkiet,
// og blir dermed testet der.
