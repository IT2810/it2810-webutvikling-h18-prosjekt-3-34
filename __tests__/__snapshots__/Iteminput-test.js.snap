// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<View
  style={
    Object {
      "borderRadius": 4,
      "borderWidth": 1,
      "flexDirection": "row",
      "marginBottom": 10,
      "paddingLeft": 10,
      "paddingRight": 10,
      "paddingTop": 25,
    }
  }
>
  <TextInput
    allowFontScaling={true}
    autoCapitalize="words"
    placeholderColor="black"
    style={
      Object {
        "borderBottomWidth": 1,
        "marginBottom": 15,
        "marginRight": 10,
        "width": 170,
      }
    }
  />
  <View
    accessible={true}
    isTVSelectable={true}
    onResponderGrant={[Function]}
    onResponderMove={[Function]}
    onResponderRelease={[Function]}
    onResponderTerminate={[Function]}
    onResponderTerminationRequest={[Function]}
    onStartShouldSetResponder={[Function]}
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "lightgrey",
        "borderRadius": 5,
        "borderWidth": 1,
        "height": 20,
        "justifyContent": "center",
        "marginBottom": 10,
        "opacity": 1,
        "width": 70,
      }
    }
  >
    <Text
      accessible={true}
      allowFontScaling={true}
      ellipsizeMode="tail"
    >
      Add item
    </Text>
  </View>
</View>
`;
